lib/java/com/google/android/material/tabs/TabLayoutMediator.java

```diff
@@ -75,17 +76,19 @@ public TabLayoutMediator(
       @NonNull TabLayout tabLayout,
       @NonNull ViewPager2 viewPager,
       @NonNull TabConfigurationStrategy tabConfigurationStrategy) {
-    this(tabLayout, viewPager, true, tabConfigurationStrategy);
+    this(tabLayout, viewPager, true, true, tabConfigurationStrategy);
   }
 
   public TabLayoutMediator(
       @NonNull TabLayout tabLayout,
       @NonNull ViewPager2 viewPager,
       boolean autoRefresh,
+      boolean smoothScroll,
```

Comment:
Can you create a new constructor that takes the smoothScroll parameter? We don't want to get rid of any existing constructor signatures.

Reply:
sure will do that.